subsys/net/l2/ieee802154/ieee802154.c

```diff
@@ -380,9 +387,12 @@ static int ieee802154_enable(struct net_if *iface, bool state)
 
 	NET_DBG("iface %p %s", iface, state ? "up" : "down");
 
+	k_sem_take(&ctx->ctx_lock, K_FOREVER);
```

Comment:
Please add empty lines: 
- before the if (unless its statement is about what has been computed before), linux code style
- after the '}' (unless it's another '}') etc..

Apply that everywhere

Reply:
Fixed here and in all other places throughout the whole change set.